monkey/agent_plugins/exploiters/wmi/src/wmi_client.py

```diff
@@ -81,6 +84,37 @@ def login(self, host: TargetHost, credentials: Credentials):
 
         self._connected = True
 
+    def _dcom_firewall_checker(self, iInterface, timeout: float):
```

Comment:
This can be static.

Reply:
This function also has nothing to do with firewalls as far as I can tell. It's just attempting another connection. Misleading name.